libs/system-admin-pages/src/types/session.ts

```diff
@@ -0,0 +1,7 @@
+declare module "express-session" {
+  interface SessionData {
+    resubmissionArtefactId?: string;
+  }
+}
+
+export {};
```

Comment:
Do we know why is this export needed?

Reply:
Removed, looks like all works fine without